share/completions/doas.fish

```diff
@@ -0,0 +1,36 @@
+
+#
+# Completion for doas https://github.com/multiplexd/doas
+# based on the sudo completions
+#
+
+function __fish_doas_print_remaining_args
+    set -l tokens (commandline -opc) (commandline -ct)
+    set -e tokens[1]
+    # These are all the options mentioned in the man page for openbsd's "doas" (in that order).
+    set -l opts a= C= L n s u=
+    argparse -s $opts -- $tokens 2>/dev/null
+    # The remaining argv is the subcommand with all its options, which is what
+    # we want.
+    if test -n "$argv"
+        and not string match -qr '^-' $argv[1]
+        echo $argv
+        return 0
+    else
+        return 1
+    end
+end
+
+function __fish_doas_no_subcommand
+    not __fish_doas_print_remaining_args >/dev/null
```

Comment:
Conditions are already silenced, so there's no real need for this redirection and therefore function in general - just `-n "not __fish_doas_print_remaining_args"` works.

Reply:
this is from the sudo completions :P I'll remove the function then.